index.html

```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
                             </div>
                         </div>
                     </section>
+                    <section class = contact-section> 
+                        <h3 class ="contact-description">
+                            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                        </h3>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
+                     </ul>
```

Comment:
- [ ] Your form section should be in a `form` element, this is good for semantic and effectivity reasons, check [this article](https://social.msdn.microsoft.com/Forums/en-US/500e1609-612d-41a5-bb5b-30b83ee40b90/form-and-div-differences?forum=aspwebforms) for more.

- It (the form) should likewise have an `action` attribute leading to a FormSpree submission, and a `method` attribute, with the value `POST` to indicate that data is been submitted to the database.

Check out [how to use form spree here!](https://www.youtube.com/watch?v=gWgzzVVcqfA)